### ISSUE TITLE
Decide on process for endorsing a chapter

### ISSUE THREAD

**Issue by dragice** [transparentstats/guidelines#74]
I added a section about this in the "Contributing to the Guidelines" wiki but nothing about the actual process, as I'm not sure how this should happen. Would be nice though, if anyone (including people who don't have a Github account and don't want to bother creating one) can easily and rapidly endorse a chapter.

**Comment by mjskay** [comment]
I agree that a lightweight way is necessary. I had suggested something like using a Google form to allow people to submit endorsements during the release process:  https://github.com/transparentstats/guidelines/wiki/Release-Process

Getting endorsements outside the release process could involve a form that we set up to generate pull requests with a person's name added to the endorsement list on whatever chapters they select or something like that...

**Comment by dragice** [comment]
Good point. Perhaps it would make sense to allow people to only endorse releases. We could explain that (and perhaps the difference between releases and stuff in-progress) at the beginning of the wiki.
